test/base-config-fonturl.js

```diff
@@ -1,29 +1,35 @@
 var tape = require('tape');
 var mjAPI = require("../lib/main.js");
-var mjVersion = require('../package-lock.json').dependencies['mathjax'].version
+var path = require('path');
 
 tape('basic configuration: check fontURL', function (t) {
     t.plan(2);
-
-    var tex = 'a';
+    
+    mjAPI.config({
+        extensions: '[test]/version.js',
+        paths: {test: path.join(__dirname,'assets/')}
+    });
```

Comment:
I think `mjAPI.start()` is needed to restart after the configuration to avoid tape failures.